cleansweep/views/admin.py

```diff
@@ -283,3 +283,18 @@ def _load_contacts(place, data):
         contacts += p.add_contacts(prows)
     db.session.commit()
     return contacts
+
+
+@app.route("/admin/settings/export", methods=['GET'])
```

Comment:
Updated. Take a look and let me know.



On Sun, Jul 17, 2016 at 9:44 PM +0530, "Anand Chitipothu" <notifications@github.com<mailto:notifications@github.com>> wrote:


In cleansweep/views/admin.py<https://github.com/AamAadmiParty/cleansweep/pull/196#discussion_r71084917>:

> @@ -283,3 +283,18 @@ def _load_contacts(place, data):
>          contacts += p.add_contacts(prows)
>      db.session.commit()
>      return contacts
> +
> +
> +@app.route("/admin/settings/export", methods=['GET'])


Are we gonna have separate endpoint for each export? I was thinking to do it just once in admin settings. Like mentioned in #182<https://github.com/AamAadmiParty/cleansweep/issues/182> <#182<https://github.com/AamAadmiParty/cleansweep/issues/182>>.
Even if we choose to have a single export of everything, it might be good to have individual exports. The right way to have single export option work without having the import the plugins is by maintaining a registry. I'll add a detailed note about this a little later. For now, could you please move that code to committees and add individual exports?

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/AamAadmiParty/cleansweep/pull/196/files/95932c4a9a342d0575a348295739bbef0d174df0#r71084917>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AEpXy-541Kp00RjTe-Tkhc886fTjr5fPks5qWlTNgaJpZM4JOPrY>.